### PR TITLE
Remove unnecessary html code

### PR DESCRIPTION
**Pull request by gpengzhi**
Our repository has 57.5% HTML code which is misleading.

### CODE REVIEW

File: forte/data/data_utils.py [hunk 1]
```diff
@@ -20,31 +20,22 @@
...
 
-# pylint: disable=unused-argument,function-redefined,missing-docstring
-
-
-def maybe_create_dir(dirname: str) -> bool:
-    r"""Creates directory if it does not exist.
 
-    Args:
-        dirname (str): Path to the directory.
+Py3 = sys.version_info[0] == 3
```
Comment (AvinashBukkittu) [inline forte/data/data_utils.py]: Where are we using this variable?

File: forte/utils/utils_io.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
```
Comment (AvinashBukkittu) [inline forte/utils/utils_io.py]: I like this `utils_io.py` file here. However, there is another similarly named `io_utils.py` in `forte/data`. I am wondering if we can consolidate the IO related functions at one place.

File: forte/data/data_utils.py (hunk 1 above)
Comment (gpengzhi) [reply forte/data/data_utils.py]: Deleted.

File: forte/utils/utils_io.py (hunk 2 above)
Comment (gpengzhi) [reply forte/utils/utils_io.py]: Delete `ensure_dir` and rename `io_utils` to `data_utils_io`

**Pull request merged**